src/converter/one/jfr/JfrReader.java

```diff
@@ -743,4 +740,9 @@ private boolean ensureBytes(int needed) throws IOException {
         buf.flip();
         return buf.limit() > 0;
     }
+
+    public long eventTimeToNanos(long time) {
+        long nanosSinceChunkStart = (time - chunkStartTicks) * SECONDS_TO_NANOS / ticksPerSec;
```

Comment:
Multipliers can be too large even for `long` type. I suggest using floating point like in other similar places.
`1e9 / ticksPerSec` can be probably extracted to a separate variable.